### ISSUE TITLE
draft not synchronized

### ISSUE THREAD

**Issue by iamdey** [CaliOpen/Caliopen#565]
sometimes it's excerpt, sometimes pi are sent when patching a draft.

btw, those two properties are read only.

2 bugs: 

* [x] frontend: edit 3 times a draft will not sync between save, so `excerpt` is outdated
* [ ] backend: api collection and specific id is not consistent, so `pi`, `external_references` ... are sent

**Comment by iamdey** [comment]
it is mostly due to an inconsistent result from api.

Here is the diff between GETs https://alpha.caliopen.org/api/v2/messages?offset=0&limit=20&discussion_id=5a8db018-fbcb-40ec-9520-9988bd6971ae and https://alpha.caliopen.org/api/v2/messages/2a659c9a-a3da-4a94-bdc3-870428d88a6e

```diff
11c11
<               "ancestors_ids": null,
---
>               "ancestors_ids": ["377c7be7-ba85-f082-b699-5b319eebeb7a@bruit-contraire.net"],
24c24
<       "parent_id": "9f1e442c-fe3f-4e7e-a171-18c6bf066f73",
---
>       "parent_id": "",
27,28c27,28
<               "contact_ids": null,
<               "label": "",
---
>               "contact_ids": [],
>               "label": "dey@alpha.caliopen.net",
34c34
<               "label": "",
---
>               "label": "dey None",
39c39,45
<       "pi": null,
---
>       "pi": {
>               "comportment": 0,
>               "context": 0,
>               "date_update": "0001-01-01T00:00:00Z",
>               "technic": 0,
>               "version": 0
>       },
```

how to reproduce : 

1. enter a discussion, write a reply and wait 5 seconds to automatically save it
2. go to timeline then go back to the discussion

Now it is impossible to PATCH

**Comment by sapiens-sapide** [comment]
Should be fix by PR #579
